hazelcast/src/test/java/com/hazelcast/config/MapAttributeConfigReadOnlyTest.java

```diff
@@ -0,0 +1,19 @@
+package com.hazelcast.config;
+
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+
+public class MapAttributeConfigReadOnlyTest {
```

Comment:
that's why there are always 2 reviewers! 

(apparently the two reviewers should be in different time-zones as well to make sure at least one of them has a normal level of caffeine)